aesara/tensor/basic.py

```diff
@@ -1079,7 +1098,7 @@ def grad(self, inp, grads):
 _nonzero = Nonzero()
 
 
-def nonzero(a, return_matrix=False):
+def nonzero(a: np.array, return_matrix: bool = False) -> Tuple:
```

Comment:
All these functions can take not only numpy arrays, but also `TensorVariables` or in some cases lists.
The type hint from line 1042 is probably a good candidate for most functions in this file.

Reply:
+1